core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -210,11 +214,11 @@ trait WhiskActionsApi extends WhiskCollectionAPI {
      * - 409 Conflict
      * - 500 Internal Server Error
      */
-    override def create(namespace: EntityPath, name: EntityName)(implicit transid: TransactionId) = {
+    override def create(user: Identity, namespace: EntityPath, name: EntityName)(implicit transid: TransactionId) = {
```

Comment:
No, it's not since the user.namespace  can be different than the namespace in this call. In particular, I think in this case `namespace` is overloaded since it contains the possible package name as well. This is a problematic use of `namespace` throughout our code that we're all aware of, but too costly to fix now...